src/Http/Http.Abstractions/src/Metadata/IDisableValidationMetadata.cs

```diff
@@ -0,0 +1,12 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Http.Metadata;
+
+/// <summary>
+/// A marker interface which can be used to identify metadata that disables validation
+/// for a specific endpoint.
```

Comment:
```suggestion
/// A marker interface that identifies metadata used to disable validation for a specific API endpoint.
```

Reply:
@tdykstra @gewarren I removed one newline. Is there a convention for when to add a newline in `///` comments?

Reply:
I think the product team has more particular rules around line length than we (content) do.

Reply:
> I think the product team has more particular rules around line length than we (content) do.

Right, random newlines, depending on the width of you editor screen :)